src/js/materia/materia.creatorcore.js

```diff
@@ -79,11 +80,8 @@ Namespace('Materia').CreatorCore = (() => {
 		_sendPostMessage('start', null)
 	}
 
-	const alert = (title, msg, type) => {
-		if (type == null) {
-			type = 1
-		}
-		_sendPostMessage('alert', { title, msg, type })
+	const alert = (msg, title = null, fatal = false) => {
```

Comment:
breaking change: creator core _alert arguments changed order to be consistent with player core